core/src/main/scala/org/apache/spark/eventhubscommon/client/AMQPEventHubsClient.scala

```diff
@@ -44,6 +44,7 @@ private[client] class AMQPEventHubsClient(
       targetEventHubNameAndPartitions: List[EventHubNameAndPartition]) = {
     val results = new mutable.HashMap[EventHubNameAndPartition, EventHubPartitionRuntimeInformation]
     try {
+      logInfo("Getting PartitionRuntimeInformation from EventHubs instance.")
```

Comment:
why it is necessary? the latency from something like `JobScheduler's start a streaming job `  to `highest offset, xxxx` is approximately the latency between these two lines